content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -165,13 +163,13 @@ await prisma.$transaction(
 )
 ```
 
-### Interactive transactions (in Preview)
+### Interactive transactions
 
 Sometimes you need more control over what queries execute within a transaction. Interactive transactions are meant to provide you with an escape hatch.
 
-**Note**: Interactive Transactions with the [Data Proxy](/data-platform/data-proxy#providing-the-connection-string-to-prisma-migrate) are in Preview in version 4.6.0.
+Interactive transactions are generally available from version 4.7.0. Version 2.29.0 introduces interactive transactions that do not yet support the [Data Proxy](/data-platform/data-proxy). In version 4.6.0, you can use interactive transactions with the [Data Proxy](/data-platform/data-proxy) in preview. 
```

Comment:
```suggestion
Interactive transactions are available in the following versions Prisma:
- Interactive transactions are generally available from version 4.7.0.
- In version 4.6, interactive transactions were available in preview.
- From version 2.29.0 to version 4.5, interactive transactions were available in preview, but did not support the [Data Proxy](/data-platform/data-proxy).  
```

Note that I said 4.6 and not 4.6.0 because we have a patch release after 4.6.0. Off the top of my head, I'm not sure if this is true for 4.5, but I've omitted the .0 to be safe.